File: JavaScript_12.18/1581 - Conversa internacional.js
/**
 * @author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
 * @copyright Josélio Júnior (Lunatic Fox) 2022
*/

const l = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
Array.from(Array(+l.shift()), _ => {
  const y = Array.from(Array(+l.shift()), _ => l.shift());
  console.log(y.every(e => e === y[0]) ? y[0] : 'ingles' );
});

File: Lua/1581 - Conversa internacional.lua
--- author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
--- copyright: Josélio Júnior (Lunatic Fox) 2022

for _ = 1, tonumber(io.read()) do
  local t, ev = {}, true
  for _ = 1, tonumber(io.read()) do
    table.insert(t, io.read())
  end
  for i = 1, #t do
    if t[1] ~= t[i] then ev = false end
  end
  print(ev and t[1] or 'ingles')
end